README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
Ok - but if i use a component which uses a utility class (pulls it in from the utility component), the whole css of the utility component (thus all the classes) are pulled in, which is unnecessary.

I guess, utility classes should then be contained in a very minimal css, which contains exactly that one utility class, so that you basically pull in a component css file which contains just ONE utility class.

That would be actually very interesting and if put to the extreme - would tend towards crazyness like: tachyonscss? (http://tachyons.io/)
...but with the difference, that every single class is exported in a single component and then you combine those to style your components, that way you get pretty minimal css stylesheets because components re-use css a lot.

But this would also mean, that i cannot longer use BEM